docs/src/tutorials/raise-alarm.md

```diff
@@ -78,6 +78,46 @@ Alarms for child devices can be raised by publishing the alarm payload to `tedge
 where the `child-device-id` is the unique device id of the child device.
 The alarm payload structure is the same, as described in the previous section.
 
+### Raising an alarm with custom fragment
+
+Thin-edge supports the creation of alarms using custom (user-defined) fragments.
+Custom fragments are supported for both the main and child devices.
+The custom fragments can be a simple json value or a complex json value.
+
+For example, an alarm with simple custom fragment
+
+Payload:
+```json
+{
+    "text": "Temperature is very high",
+    "time": "2021-01-01T05:30:45+00:00",
+    "details": "A custom alarm info"
+}
+```
+
+For example, an alarm with complex custom fragments
+
+Payload:
+```json
+{
+    "text": "Temperature is very high",
+    "time": "2021-01-01T05:30:45+00:00",
+    "someOtherCustomFragment": {"nested":{"value": "extra info"}}
+}
+```
+
+> Note: Other than `text` and `time` fields, all the other fields are considered as custom fragments.
+
+### Raising an alarm with empty payload
+
+Alarms can also be raised with a fixed message payload for `thin-edge` as well for the `child` devices.
+The payload will be an `empty json object` message as below
```

Comment:
Two critical points are missing here. One needs to:

1. Stress the difference between an empty string message payload `""`, that clears the alarm, and an empty json payload `{}` that raises an alarm with default values.
2. Tell what are the default values for the timestamp and the text message (if any) of an alarm created by `{}`.

Reply:
I have updated the document by adding more details to empty ` the""` message and empty json `{}` message.

Reply:
If you are mentioning that its for `{}` case then there is already a section here `### Raising an alarm with empty json payload`